org.opentosca.bus.management.service.impl/src/org/opentosca/bus/management/service/impl/collaboration/RequestReceiver.java

```diff
@@ -0,0 +1,583 @@
+                                    Activator.producer.sendBodyAndHeaders("direct:SendMQTT", reply, headers);
+                                } else {
+                                    RequestReceiver.LOG.error("No invocation plug-in found for invocation type: {}",
+                                                              invocationType);
+                                }
+                            } else {
+                                RequestReceiver.LOG.error("No invocation type specified for the IA!");
+                            }
+                        } else {
+                            RequestReceiver.LOG.error("Body contains no IAInvocationRequest. Aborting operation!");
+                        }
+                    } else {
+                        RequestReceiver.LOG.error("Collaboration message contains no body. Aborting operation!");
+                    }
+                } else {
+                    RequestReceiver.LOG.error("Message body has invalid class: {}. Aborting operation!",
+                                              message.getBody().getClass());
+                }
+            } else {
+                RequestReceiver.LOG.error("Request does not contain all needed header fields to send a response. Aborting operation!");
+            }
+        } else {
+            RequestReceiver.LOG.debug("Request is directed to another OpenTOSCA Container. Ignoring request!");
+        }
+    }
+
+    /**
+     * Get the header fields that are needed to respond to a request as Map.
+     *
+     * @param message the request message
+     * @return the Map containing the header fields for the response if the needed header fields are
+     *         found in the request message, <tt>null</tt> otherwise
+     */
+    private Map<String, Object> getResponseHeaders(final Message message) {
+
+        // extract header field
+        final String broker = message.getHeader(MBHeader.MQTTBROKERHOSTNAME_STRING.toString(), String.class);
+        final String replyTopic = message.getHeader(MBHeader.REPLYTOTOPIC_STRING.toString(), String.class);
+        final String correlation = message.getHeader(MBHeader.CORRELATIONID_STRING.toString(), String.class);
+
+        // reply is only possible if all headers are set
+        if (broker != null && replyTopic != null && correlation != null) {
+
+            // add the header fields to the header map and return it
+            final Map<String, Object> headers = new HashMap<>();
+            headers.put(MBHeader.MQTTBROKERHOSTNAME_STRING.toString(), broker);
+            headers.put(MBHeader.MQTTTOPIC_STRING.toString(), replyTopic);
+            headers.put(MBHeader.CORRELATIONID_STRING.toString(), correlation);
+
+            return headers;
+        } else {
+            // header fields are missing and therefore no response possible
+            return null;
+        }
```

Comment:
invert and dedent. the inverted condition is also generally easier to reason about and won't need a comment